meta_request/lib/meta_request/log_interceptor.rb

```diff
@@ -3,33 +3,33 @@
 module MetaRequest
   module LogInterceptor
 
-    def debug(message = nil, &block)
-      push_event(:debug, message)
+    def debug(*message)
+      push_event(:debug, message.join(' '))
```

Comment:
i'm not sure this is the right way to go - there's no guarantee that `message.join(' ')` will do the right thing with multiple params. why not just change the signature to `(message = nil, *args, &block)`?
this should handle your use case.